src/objdump.py

```diff
@@ -424,7 +428,15 @@ def fn(pat: Match[str]) -> str:
 def objdump(
     o_filename: str, arch: ArchSettings, *, stack_differences: bool = False
 ) -> List[Line]:
-    output = subprocess.check_output(arch.objdump + [o_filename])
+    executable = None
+    for cand in arch.executable:
+        if shutil.which(cand):
+            executable = cand
+            break
+    if executable is None:
+        raise Exception(f"Could not find objdump executable for {arch.name}.")
```

Comment:
Add some info about which executable names are searched for to this message, because I would definitely expect this to show up in practice.